package.json

```diff
@@ -50,7 +50,7 @@
         "@types/find-config": "^1.0.1",
         "keyv-file": "^0.2.0",
         "koa-cors": "^0.0.16",
-        "lerna": "^4.0.0",
+        "lerna": "^5.6.2",
```

Comment:
You updated the version but you package-lock.json is out of date.
Please run `npm i` and commit the changes for consistency and to unblock build and tests